automation/src/test/scala/org/broadinstitute/dsde/test/api/Submission.scala

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.dsde.test.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.service.test.RandomUtil
+import org.broadinstitute.dsde.workbench.service.Rawls
+import org.scalatest.Matchers._
+import org.scalatest.concurrent.Eventually
+import org.scalatest.time.{Minutes, Seconds, Span}
+
+
+object Submission extends LazyLogging with Eventually with RandomUtil {
+
+  def getSubmissionStatus(billingProject: String, workspaceName: String, submissionId: String)(implicit token: AuthToken): String = {
+    val (status, workflows) = Rawls.submissions.getSubmissionStatus(billingProject, workspaceName, submissionId)
+    logger.info(s"Submission $billingProject/$workspaceName/$submissionId status is $status")
```

Comment:
Double log here? `Rawls.submissions.getSubmissionStatus` should already log this right? Or is this so you can see what the `status` actually is?

Reply:
`Rawls.submissions.getSubmissionStatus` is not logging status from response. so I'm logging it here.